content/docs/next-release/operator.md

```diff
@@ -334,13 +334,61 @@ The default create-schema job uses `MODE=prod`, which implies a replication fact
 
 ### Elasticsearch storage
 
+By default Elasticsearch storage does not require to run any initialization job. However Elasticsearch
```

Comment:
"require any initialization job to be run"
"storage requires a cron job to be run to clean...."